src/Illuminate/Database/Eloquent/Collection.php

```diff
@@ -280,6 +280,21 @@ public function map(callable $callback)
         }) ? $result->toBase() : $result;
     }
 
+    /**
+     * Replicates every item in the collection.
+     *
+     * @param  array|null  $except
+     * @return static
+     */
+    public function replicate(array $except = null)
+    {
+        $items = $this->map(function ($model) use ($except) {
```

Comment:
Can we just return here? The resulting collection from the `map` should be a new instance of the Eloquent collection (assuming all the returned objects are eloquent collections), so the following `new static` is redundant I believe.